### DISCUSSION TITLE
Is there a way to concatenate a string literal with a value pulled during web scraping (json dot notation)

### DISCUSSION THREAD

**Discussion by Bhuemann** [FreshRSS/FreshRSS#6561] [Q&A]
So I am trying to create a feed from Netflix job postings. I have pulled their API call and have been able to create a feed from it using an arbitrary choice for URL.

URL: https://jobs.netflix.com/api/search?q=software&location=Remote%2C%20United%20States

However the response here does not return the full job URL, just the slug. So I would like to concatenate the base address with the slug something like so:

`strcat("https://jobs.netflix.com/jobs/", external_id)`
Or
`"https://jobs.netflix.com/jobs/" + external_id`

I've tried both but they don't seem to work and I couldn't find any documentation on this. Is this possible?

Note: I have noticed that if I just put the _external_id_ in the url field, the feed items generated are prefixed with "https://jobs.netflix.com/api/" which I don't know where this value comes from and if it can be changed to the correct base address

![image](https://github.com/FreshRSS/FreshRSS/assets/9254008/12295f1e-ce10-4a3b-8090-acbd9462b116)

**Comment by Frenzie** [comment]
Have you tried xml:base?

**Reply by Bhuemann** [reply]
Could you give a little bit more context by what you mean here?

**Reply by Frenzie** [reply]
Pardon, I hadn't noticed you were talking about the JSON scraping. xml:base specifies the base URI in a normal feed. If the equivalent can't be done in the JSON functionality I'd consider that a missing feature.
